spec/models/user_spec.rb

```diff
@@ -37,7 +37,7 @@
       let!(:user) { create(:user) }
 
       before do
-        user.accounts.create({ name: 'Emprego CLT', active: false }, CltWorkerAccount)  
+        user.accounts.create({ name: 'Emprego CLT', active: false }, CltWorkerAccount)
```

Comment:
Line is too long. [86/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.